packages/core/src/webapp/util/ascii-to-table.ts

```diff
@@ -362,7 +442,7 @@ export const formatTable = (entityType: string, lines: Pair[][]): Table => {
       .map(({ key, value: column, valueDom, css }, colIdx) => ({
         key,
         tag: (idx > 0 && tagForKey[key]) || undefined,
-        onclick: false,
+        onclick: isKube && colIdx + 1 === nameColumnIdx && onclick, // see the onclick comment: above ^^^; +1 because of slice(1)
```

Comment:
Added `isKube` to `onclick` to white-list `oc`. All the other changes are from reverting commit 39500ba350b7896c831d88d4424affc376a9ed9d .